spec/factories/tag_field_factory.rb

```diff
@@ -1,19 +1,21 @@
 FactoryBot.define do
-  factory :tag_field, class: "DocumentType::TagField" do
+  factory :tag_field, class: "DocumentType::OrganisationsTagField" do
```

Comment:
Thanks for pointing this out in the commit - I missed it when I was reading through. For now, we should dispense with the factory and use the same approach we did for [the original PR on this work](https://github.com/alphagov/content-publisher/pull/1705/commits/b7033fee13ab2c1f4e96ab69e4c99d15c7882893#diff-7ebb5ee02d495684dcb59a514e54bb8aR19).